src/data/components.json

```diff
@@ -274,7 +274,7 @@
     "Footer": {
       "url": "/components/footer",
       "designLink": "https://ibm.box.com/s/nsbccv6jigfw78uwq3r0242qks1x1f87",
-      "description": "The footer component is a section that sits at the bottom of any IBM.com page, and acts as the catch-all section that helps users navigate and provides corporate level general and legal information. The footer is required on all pages on IBM.com.",
+      "description": "The Footer is required on all IBM.com pages, it is placed at the bottom of a page and acts as the catch-all section that helps users navigate and provides corporate level general and legal information.",
```

Comment:
In all our other documentation when referencing components/elements, we don't use title case. Should we try to stay consistent here too?